bits_test.go

```diff
@@ -6,7 +6,7 @@ import (
 	"time"
 )
 
-func TestGetBitsLeaderboard(t *testing.T) {
+func TestClient_GetBitsLeaderboard(t *testing.T) {
```

Comment:
Why are we changing this? Not against the change, just want to understand if this is new Go convention that I'm not aware of.

Reply:
It's a convention at least by Goland IDE. When I'm creating test methods, it's actually suggesting a method name based on the missing method and the convention you see above. I was curious, so I checked to see if it would suggest similar for an existing test method, and it did.

Personally I don't have a preference, I just updated it here for consistency.